docs/src/getting-started/register.md

```diff
@@ -70,38 +75,41 @@ phrase and then convert it into a seed secret we can use:
 	# Store the seed on the filesystem, or secure configuration system
 	```
 
-!!! important 
+!!! important
 	Remember to store the seed somewhere (file on disk, registry, etc)
 	because without it, you will not have access to the node, and any
 	funds on the node will be lost forever! We mean it when we say _you're
 	the only one with access to the seed_!
 
+## Registering the node
 
-Next we instantiate an mTLS identity we will use to authenticate with
-Greenlight. Since we haven't registered our node with the service yet,
-we will use a [dummy key][auth], that allows us to talk to the 
-[`Scheduler`][scheduler] but will not allow us to talk to any other service.
-No worries, once we register the node we will get a valid certificate 
-to authenticate.
+In the next step we instantiate an mTLS identity that can be used to 
+authenticate to Greenlight. We will use your [developer identity][certs]
+which can be used to register a node.
 
 === "Rust"
 	```rust
 	use gl_client::tls::TlsConfig;
 
-	let tls = TlsConfig::new();
+	# Creating a new `TlsConfig` object using your developer certificate
+	# - cert: contains the content of `client.crt`
+	# - key: contains the content of `client-key.pem`
+	let tls = TlsConfig::new().identity(cert, key);
```

Comment:
I agree, executable examples would be great.

It would require some boilerplate (reading and writing certificates) that make the it harder to spot where the action is happening. 

We could implement a utility method called `identity_from_path` which points to the location of `client.crt` and `client-key.pem`.

In this case the code would become

```
let tls = TlsConfig::new().identity_from_path("./path-to-dev-certs/")
```

Reply:
Absolutely agree, we should have copy-pasteable code examples, and some helpers can be useful to get there without too much clutter. Let's be minimal with the variants though, if there is a simple way to do something it should be the one described, and we should stick with that. Alternatives can be described in deep dives that talk specifically about the scenario where it is needed.

Example: let's remove the compiled-in certificate in lieu of the runtime-provided one, as the latter is more widely applicable.